crates/turbopack-ecmascript/src/chunk/mod.rs

```diff
@@ -977,6 +977,8 @@ impl Asset for EcmascriptChunk {
                 hasher.write_value(id.await?);
                 need_hash = true;
             }
+            // Marker to end evaluate
+            hasher.write_value(true);
```

Comment:
Should this have written a length header instead of a bool footer?

Reply:
Both `evaluate` and `omit` could write length headers (or `0` when N/A).